### DISCUSSION TITLE
MFA on qualitative table

### DISCUSSION THREAD

**Discussion by jkevink** [MaxHalford/prince#166] [Q&A] [answered]
Was wondering in MFA there are options to declare if the table is quantative or qualitative (or mixed) in prince?

**Comment by MaxHalford** [comment, marked as answer]
Hey there. MFA in Prince only works with numeric data. There might be an extension possible for qualitative data, but I'm not aware of it.

**Reply by jkevink** [reply]
Yeah it's quite limited in Python, thanks a lot for the prompt response!
